circle.yml

```diff
@@ -4,6 +4,11 @@ machine:
   services:
     - redis
 dependencies:
+  pre:
+    - curl -L -o google-chrome.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+    - sudo dpkg -i google-chrome.deb
+    - sudo sed -i 's|HERE/chrome\"|HERE/chrome\" --disable-setuid-sandbox|g' /opt/google/chrome/google-chrome
```

Comment:
I assume we're adding the `disable-setuid-sandbox` because we need that option for our tests to work properly.  Can you add a comment here?